apps/default/loki/config.yaml

```diff
@@ -13,30 +11,41 @@ common:
   ring:
     kvstore:
       store: inmemory
-
 schema_config:
   configs:
     - from: 2020-10-24
-      store: boltdb-shipper
+      store: tsdb
       object_store: filesystem
-      schema: v11
+      schema: v13
       index:
         prefix: index_
         period: 24h
-
 query_scheduler:
   max_outstanding_requests_per_tenant: 4096
 frontend:
   max_outstanding_per_tenant: 4096
 query_range:
   parallelise_shardable_queries: true
 limits_config:
-  split_queries_by_interval: 15m
-  max_query_parallelism: 32
+  allow_structured_metadata: true # This is required to allow Loki to handle OpenTelemetry data https://grafana.com/docs/loki/latest/send-data/otel/#loki-configuration
+  reject_old_samples: false
+  reject_old_samples_max_age: 365d
```

Comment:
I am not sure we want these. This setting is really only needed for the pushing old historical data (seed-dora)